src/subcommand/wallet/create.rs

```diff
@@ -1,8 +1,97 @@
-use super::*;
+use {
+  super::*,
+  bitcoin::secp256k1::{rand::RngCore, All, Secp256k1},
+  bitcoin::{
+    util::bip32::{ChildNumber, DerivationPath, ExtendedPrivKey, Fingerprint},
+    Network,
+  },
+  bitcoincore_rpc::bitcoincore_rpc_json::{ImportDescriptors, Timestamp},
+  miniscript::descriptor::{Descriptor, DescriptorSecretKey, DescriptorXKey, Wildcard},
+};
+
+#[derive(Debug, Parser)]
+pub(crate) struct Create {
+  #[clap(long, default_value = "ord", help = "Create wallet with <NAME>")]
+  pub(crate) name: String,
+}
+
+impl Create {
+  pub(crate) fn run(&self, options: Options) -> Result {
+    if !(self.name == "ord" || self.name.starts_with("ord-")) {
```

Comment:
We should add a test for this, that if you pass a bad value to `create` it refuses to run.